tests/ImageSharp.Tests/Colorspaces/Conversion/RgbAndHslConversionTest.cs

```diff
@@ -31,6 +50,8 @@ public class RgbAndHslConversionTest
         [InlineData(0, 1, .5F, 1, 0, 0)]
         [InlineData(120, 1, .5F, 0, 1, 0)]
         [InlineData(240, 1, .5F, 0, 0, 1)]
+        //[Theory]
```

Comment:
Is this WIP stuff now here?

Reply:
It was just added to generate all the required info for https://github.com/tonerdo/coverlet/issues/700

I'll remove it